src/vivarium_gates_iv_iron/constants/data_values.py

```diff
@@ -6,3 +8,33 @@
 POSTPARTUM_DURATION_RATIO = POSTPARTUM_DURATION_DAYS / TOTAL_POSTPARTUM_DAYS
 
 
+class _HemoglobinDistributionParameters(NamedTuple):
+    XMAX: int = 220
+    EULERS_CONSTANT: float = 0.57721566490153286060651209008240243104215933593992
```

Comment:
FYI, you could use `sympy.EulerGamma` instead of defining it Euler's Constant here like this. I think I'd prefer that